Sources/VaporSQLite/Provider.swift

```diff
@@ -42,10 +24,22 @@ public final class Provider: Vapor.Provider {
             throw Error.pathMissing
         }
         
-        try self.init(path: path)
+        self.path = path
         
     }
     
+    public func boot(_ drop: Droplet) {
+        
+        guard let driver = try? SQLiteDriver(path: drop.workDir+self.path) else {
```

Comment:
We should support both absolute and relative here.

If the path has a leading `/`, assume absolute. Otherwise, assume relative.